api/src/locale/en/messages.json

```diff
@@ -287,16 +336,61 @@
     "origin": [
       [
         "src/schema/index.js",
-        90
+        105
+      ]
+    ]
+  },
+  "Heated floor area in m2": {
+    "translation": "",
+    "origin": [
+      [
+        "src/schema/index.js",
+        107
       ]
     ]
   },
-  "The EnerGuide Rating calculated for this evaluation": {
+  "The EnerGuide Rating calculated for this evaluation, in GJ": {
     "translation": "",
     "origin": [
       [
         "src/schema/index.js",
-        92
+        109
+      ]
+    ]
+  },
+  "The EnerGuide Rating calculated for this evaluation, actual rating": {
```

Comment:
I should have mentioned before, this is a deprecated field, so newer evaluations will have this be NULL, we should probably get that across somehow